.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
It is possible to get and Error name in both lines, only second line is for "Error", first one should be left alone for operations, we want the user to be able to see what he did wrong in the operation that resulted in an error
![image](https://github.com/user-attachments/assets/474e516c-11a3-4d25-ab1f-f669eeebc6de)

This reproduction:
9999999999*999999999= and then * and starting typing again 9999 results in Error in both lines

Reply:
I actually did similar thing when I did 0-1111111111 - 9999999999 few times

![image](https://github.com/user-attachments/assets/2a9a5619-f057-402e-8206-ab32d3bc9b5b)